lib_test/device_add_pci.json

```diff
@@ -0,0 +1 @@
+{ "execute": "device_add", "arguments":{"driver":"Xen-pci-passthrough","id":"pci","bus":"0", "hostaddr":"some_address", "permissive":true}}
```

Comment:
I think we should use a valid example here that Qemu would accept.
It should be possible to see the actual QMP command that `xl` sends if we turn on debugging.
I don't think bus is correct here, since we know its not changeable, and we are missing "addr".